Example/Example/Shared/Configuration.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Configuration.swift
+//  Example
+//
+//  Created by Ryan Davis on 4/8/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Foundation
+import os.log
+
+private extension TimeInterval {
+  static var oneDay: Self { 60 * 60 * 24 }
+}
+
+private enum UserDefaultsKey {
+  static var configuration: String { #function }
+  static var lastFetchDate: String { #function }
```

Comment:
😮 does `#function` work here as the name of the getter?

Reply:
Yeah, cool hey!